service/history/queues/dlq_writer.go

```diff
@@ -86,7 +87,16 @@ func (q *DLQWriter) WriteTaskToDLQ(
 	if err != nil {
 		return fmt.Errorf("%w: %v", ErrSendTaskToDLQ, err)
 	}
-	metrics.DLQWrites.With(q.metricsHandler).Record(1, metrics.TaskCategoryTag(task.GetCategory().Name()))
+	// "passive" means the namespace is in standby mode and only replicates data
+	namespaceState := "passive"
+	if isNamespaceActive {
+		namespaceState = "active"
+	}
```

Comment:
Is NamespaceStateTag used in other places and do we have any constant defined for its values? It will be ideally if all usages are using the same value.